client/my-sites/sites/sites.jsx

```diff
@@ -47,7 +47,7 @@ export const Sites = React.createClass( {
 
 		// Filter out sites with no upgrades on particular routes
 		if ( /^\/domains/.test( path ) || /^\/plans/.test( this.props.sourcePath ) ) {
-			return this.props.isSiteUpgradeable( site.ID );
+			return site.isSiteUpgradeable !== false;
```

Comment:
Sure, I'd go that way, but I was wondering: is there a case where upgradeable state can't be determined for a WordPress.com site? For example, if a site request fails to complete.